app/models/placements/schools/placements/build/placement.rb

```diff
@@ -61,6 +73,13 @@ def valid_additional_subjects?
     end
   end
 
+  def valid_year_group?
+    return true if school.phase != "Primary" || year_group.present?
+
+    errors.add(:year_group, :invalid)
+    false
+  end
```

Comment:
If we use the enum, then we should be able to pass `validate: true` and it'll check that it's one of the accepted values.